rust/tools/authorization-logic/test_inputs/typeDeclarations

```diff
@@ -0,0 +1,7 @@
+.decl someFact(x : symbol)
+.decl someOtherFact(x : symbol, y : number)
+.decl thirdFact(p : principal)
```

Comment:
I think technically since this comment is on the frontend of the auth logic language it does not matter because the backend could still emit `Principal`... but perhaps it's best to keep it stylistically consistent anyway.